src/entity/sources/SourceCategory.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, PrimaryColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+export class SourceCategory {
+  @PrimaryColumn({ type: 'text' })
```

Comment:
uuid maybe better for pk, we us it on more places recently?

Reply:
Much better.